gdcdictionary/schemas/exposure.yaml

```diff
@@ -436,8 +613,6 @@ properties:
     minimum: 1900
 
   tobacco_smoking_status:
-    $ref:
-      - "_terms.yaml#/tobacco_smoking_status/common"
     enum:
       - "1"
```

Comment:
I had thought the number enums were being deprecated, do not see new enums listed here for "tobacco_smoking_status"? Also not sure why $ref to _terms.yaml file was removed, I think it is still needed?

Reply:
added ref back in